governance/votes/Executive vote - October 17, 2024.md

```diff
@@ -0,0 +1,287 @@
+- LSE-MKR-A will be added to the Debt Ceiling Instant Access Module with the following parameters:
+  - [`line`]((https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4.1_Maximum_Debt_Ceiling_(line)-6f1a913d-9436-4b70-816b-e317672737d6|57eaf45219bea3b430c268bb)): **20 million DAI**.
+  - [`gap`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4.2_Target_Available_Debt_(gap)-c1e17312-3744-4360-81f4-03564f0dd09d|57eaf45219bea3b430c268bb): **5 million DAI**.
+  - [`ttl`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.4.3_Ceiling_Increase_Cooldown_(ttl)-58f5315f-938f-464c-b2d2-f611145d469a|57eaf45219bea3b430c268bb): **16 hours**.
+- LSE-MKR-A will be added to the ilk registry with the provided `name` and `symbol` and the new ilk class of **7**.
+- MCD_SPOT, CLIPPER_MOM, OSM_MOM, MCD_END will use the new MKROSM (PIP_MKR) to assess the collateral price.
+- PIP_MKR will be added to the OSM_MOM.
+- Lockstake Clipper will be configured with the following parameters:
+  - [`buf`](ttps://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.2_Auction_Price_Multiplier_(buf)-3ce44221-fe39-4904-9f86-0a1996cea8c6|57eaf45219bea3b430c2a59a): **1.20**.
+  - [`tail`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.4_Max_Auction_Duration_(tail)-f8de0fc0-e452-4fc6-b67c-6da15df828f0|57eaf45219bea3b430c2a59a): **6,000 seconds**.
+  - [`cusp`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.3_Max_Auction_Drawdown_(cusp)-083ada9d-8d45-46ae-80d2-a3bb706e1e77|57eaf45219bea3b430c2a59a): **0.40**.
+  - [`chip`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.5_Proportional_Kick_Incentive_(chip)-afecb109-75a7-4289-997e-2a6f3d3bea1e|57eaf45219bea3b430c2a59a): **0.1%**.
+  - [`tip`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.6_Flat_Kick_Incentive_(tip)-1c13687a-2dad-4e86-9940-024dc8f17a24|57eaf45219bea3b430c2a59a): **300 DAI**.
+  - `stopped`: **0**.
+  - [`tolerance`](https://sky-atlas.powerhouse.io/#A.1.8.2.2.4.1_Liquidations_Circuit_Breaker_Exception_Price_Tolerance-bcccd6b0-ad61-47c6-94f1-93447ba39102|0db30758e055352ce6eb): **0.5**.
+- StairstepExponentialDecrease calc contract will be configured with the following parameters:
+  - [`cut`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.1_Auction_Price_Function_(cut)-b6e47d05-addf-4371-ad95-90582da7c0f1|57eaf45219bea3b430c2a59a6013): **0.99**.
+  - [`step`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.2_Auction_Price_Function_(step)-7d58c961-43ff-4157-80aa-8a92f577ec12|57eaf45219bea3b430c2a59a6013): **60 seconds**.
+- REWARDS_LSMKR_USDS will be added to the Seal Engine as the first farm.
+- The LockstakeEngine contract will be authorized to access the `vat`.
+- The LockstakeClipper contract will be authorized to access the `vat` and the LockstakeEngine.
+- CLIPPER_MOM, MCD_DOG, and MCD_END will be authorized to access LockstakeClipper.
+- New [Chainlog](https://chainlog.sky.money) keys LOCKSTAKE_MKR, LOCKSTAKE_ENGINE, LOCKSTAKE_CLIP, and LOCKSTAKE_CLIP_CALC will be added.
+
+#### Early Bird Rewards Multisig Funding
```

Comment:
```suggestion
### Early Bird Rewards Multisig Funding
```
Imo, this is not related to the Seal Engine Initialization

Reply:
Fair assessment. I will change the Executive Summary to reflect this as well.